test/Utilities/intervals_fhicl_test.cc

```diff
@@ -46,8 +47,7 @@ fhicl::Table<Config> validateConfig(fhicl::ParameterSet const& pset) {
 // -----------------------------------------------------------------------------
 template <typename Config>
 fhicl::Table<Config> validateConfig(std::string const& configStr) {
-  fhicl::ParameterSet pset;
-  pset = fhicl::ParameterSet::make(configStr);
+  auto pset = fhicl::ParameterSet::make(configStr);
   return validateConfig<Config>(pset);
```

Comment:
This is possibly a residual from debugging; feel free to collapse it in a single line (unless I am missing something and it's not actually possible):
```suggestion
  return validateConfig<Config>(fhicl::ParameterSet::make(configStr));
```

Reply:
Done.